proto/ibc/core/port/v1/query.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package ibc.core.port.v1;
+
+option go_package = "github.com/cosmos/ibc-go/modules/core/05-port/types";
+
+import "ibc/core/channel/v1/channel.proto";
+
+// Query provides defines the gRPC querier service
+service Query {
+  // Port queries an IBC Port.
```

Comment:
I think we should add some docs here for relayer users

// CounterpartyVersion takes a port-id and routes to the respective application module VersionNegotiation implementation

Reply:
Yep, definitely! 👍  Planned on improving this when naming is settled

Reply:
We can add information about this gRPC in our migration docs